data/seeds/organizations.yml

```diff
@@ -519,3 +519,15 @@
   twitter_handle: Codesthq
   locations:
   - address: ul. Fabryczna 20A, 31-553 Kraków, Poland
+- name: Stitch Fix 
+  type: business
+  url: https://www.stitchfix.com 
+  twitter_handle: stitchfix
+  locations:
+  - address: 1 Montgomery Tower, San Francisco, CA 94104, USA
+- name: Bigger Pockets
+  type: business
+  url: https://www.biggerpockets.com 
+  twitter_handle: biggerpockets
+  locations:
+    - address: 3344 Walnut St, Denver, CO 80205, USA
```

Comment:
```suggestion
  - address: 3344 Walnut St, Denver, CO 80205, USA
```

Reply:
🤦‍♀

Reply:
it was my fault! made a dodgy suggestion 🤦‍♂